### PR TITLE
Add instance-norm as a flag (still defaults to batch-norm)

### PR DESCRIPTION
**Pull request by noodlefrenzy**

### CODE REVIEW

File: cyclegan.py [hunk 1]
```diff
@@ -226,22 +243,21 @@ def generator(image, reuse=False, name="generator"):
         def residual_block(x, dim, ks=3, s=1, name='res', reuse=reuse):
             p = int((ks - 1) / 2)
             y = tf.pad(x, [[0, 0], [p, p], [p, p], [0, 0]], "REFLECT")
-            y = batch_norm(conv2d(y, dim, ks, s, padding='VALID', name=name + '_c1', reuse=reuse), name + '_bn1',
-                           reuse=reuse)
+            y = conv2d(y, dim, ks, s, padding='VALID', name=name + '_c1', reuse=reuse)
+            y = do_norm(y, norm, name + '_1', reuse)
```
Comment (olgaliak) [inline cyclegan.py]: residual blocks by default should not have batch norm

**Comment by noodlefrenzy** [comment]
@olgaliak I've updated, should address your requests.
